starter-code/index.html

```diff
@@ -0,0 +1,206 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="css/bootstrap.min.css">
+
+    <!-- My Own CSS -->
+    <link rel="stylesheet" href="css/styles.css">
+
+    <title>clone - revera</title>
+  </head>
+
+  <!-- top header -->
+  <div class="container-fluid my-nav-container">
+    <nav id="my-navigation" class="navbar navbar-expand-lg navbar-light bg-light">
+      <div class="space col-md-1"></div>
+      <h1><a id="my-company-name" class="navbar-brand" href="#">REVERA</a></h1>
+      <div class="space col-md-3"></div>
+      <div class="collapse navbar-collapse" id="navbarNavDropdown">
+        <ul class="navbar-nav">
+          <li class="nav-item active">
+            <a id="my-home-link" class="nav-link my-highlight" href="#">Home <span class="sr-only">(current)</span></a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link my-highlight" href="#">About</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link my-highlight" href="#">Sample page</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link my-highlight" href="#">Portfolio</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link my-highlight" href="#">Blog</a>
+          </li>
+          <li class="nav-item dropdown">
+            <a class="nav-link my-highlight" href="#" id="navbarDropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+              Parent Page
+              <span class="sf-sub-indicator"> »</span>
+            </a>
+            <div class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink">
+              <a class="dropdown-item" href="#">Full page</a>
+              <a class="dropdown-item" href="#">Child page 1</a>
+              <a class="dropdown-item" href="#">Child Page 2</a>
+            </div>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link my-highlight" href="#">Contact</a>
+          </li>
+        </ul>
+      </div>
+    </nav>
+  </div>
+  <!-- end top header -->
+
+  <!-- slidebox container -->
+  <!-- temporary placeholder -->
+  <div class="slidebox">
```

Comment:
for this div you could have used a carousel from bootstrap to get the effect from the page you were cloning.